### ISSUE TITLE
Merge params and queryParams

### ISSUE THREAD

**Issue by guilhermeaiolfi** [guilhermeaiolfi/flx#1]
In the enter hood merge params like:

``` javascript
var params = jQuery.extend(ctx.transition.routes.queryParams, current_route.params);
```

jQuery is already available. But maybe lodash should be considered.

**Comment by silentworks** [comment]
I saw this and was wondering why not use lodash `_.deepClone` method instead? I might not have/want jQuery in my project at all.

**Comment by guilhermeaiolfi** [comment]
Yes. Using ```_.deepClone``` there and remove the ```jQuery.on/off`` for the ```hashchange``` event we could remove the jQuery dep.
